parser-typechecker/src/Unison/Server/Doc.hs

```diff
@@ -273,7 +275,7 @@ renderDoc pped terms typeOf eval types tm = eval tm >>= \case
                         full tm typ =
                           formatPretty (TermPrinter.prettyBinding ppe name (Term.ann() tm typ))
                     pure (DO.UserObject (Src folded (full tm typ)))
-              Term.RequestOrCtor' r _ | Set.notMember r seen -> (:acc) <$> goType r
+              Term.RequestOrCtor' (view ConstructorReference.reference_ -> r) | Set.notMember r seen -> (:acc) <$> goType r
```

Comment:
I think the trade off is worth it TBH; 
we should try to reduce cases where we strip the references off of things like this anyways since it's losing any type-info about the references.